Taarafo.Core/Controllers/CommentsController.cs

```diff
@@ -0,0 +1,53 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc;
+using RESTFulSense.Controllers;
+using System.Threading.Tasks;
+using Taarafo.Core.Models.Comments;
+using Taarafo.Core.Models.Comments.Exceptions;
+using Taarafo.Core.Services.Foundations.Comments;
+
+namespace Taarafo.Core.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class CommentsController : RESTFulController
+    {
+        private readonly ICommentService commentService;
+
+        public CommentsController(ICommentService commentService) =>
+            this.commentService = commentService;
+
+        [HttpPost]
+        public async ValueTask<ActionResult<Comment>> CommentCommentAsync(Comment comment)
```

Comment:
PostCommentAsync than CommentCommentAsync

Reply:
@ShriHumrudha I corrected this

Reply:
I still don't see the change here.